docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```

Comment:
Is this file needed?

Reply:
I think it's how most people build docs on there machines, I'll do a quick check though.